Rag-and-texttosql/rag_sql_app_upload.py

```
@@ -0,0 +1,258 @@
+import streamlit as st
+import os
+import tempfile
+from llama_index.core import VectorStoreIndex, ServiceContext
```

Ruff (0.8.2)

`llama_index.core.ServiceContext` imported but unused

Remove unused import: `llama_index.core.ServiceContext`

(F401)